packages/server/utils/analytics/amplitude/AmplitudeAnalytics.ts

```diff
@@ -0,0 +1,29 @@
+import {init, track} from '@amplitude/analytics-node'
+import {AnalyticsEvent} from '../analytics'
+import {getUserById} from '../../../postgres/queries/getUsersByIds'
+import PROD from '../../../PROD'
+
+const {AMPLITUDE_WRITE_KEY} = process.env
+
+/**
+ * Wrapper for amplitude providing a more typesafe interface
+ */
+export class AmplitudeAnalytics {
+  constructor() {
+    init(AMPLITUDE_WRITE_KEY || 'x', {
```

Comment:
-1 Why `'x'`? If it's just to satisfy the types, I would expect `''`. If `'x'` has any special meaning, please add a code comment.

Reply:
I had copied the initialization logic from `segmentIo.ts` without thinking too much about it, which included the ` || 'x'` logic.

After reviewing, we have no real need to initialize the SDK at all if we don't have a write key, so I added an early exit in the constructor if the env var is not set.